evaluation_script/main.py

```diff
@@ -71,7 +59,7 @@ def populate_index_map(infile):
             break
     return index_map
 
-def calculate_base_metrics(infile, truth):
+def calculate_base_metrics(infile, truth, documents_with_ground_truth, query_level_metrics):
     """Processes a prediction file and calculates base metrics.
```

Comment:
Update the docstring to describe the return value. Currently the docstring indicates that the global `query_level_base_metrics` dictionary is updated. This is of course no longer true. Instead this dictionary is created and returned along with the `global_metrics` object.

Reply:
Addressed in latest update